platform/security/handler/security-handler-pki/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -43,4 +43,17 @@
              ranking="75">
     </service>
 
+    <bean id="crlChecker" class="org.codice.ddf.security.handler.pki.CrlChecker" />
```

Comment:
So we're basically creating an instance of `CrlChecker` here that's different from the one used by the `AbstractPKIHandler`, right? Would there be a way to make sure we only have one instance? I'd hate for them to somehow become out-of-sync since they're not stateless.